calico-cloud/threat/web-application-firewall.mdx

```diff
@@ -52,6 +52,16 @@ You simply enable WAF in Manager UI, and determine the services that you want to
 - WAF is not supported for host-networked client pods
 - When selecting and deselecting traffic for WAF, active connections may be disrupted
 
+:::note warning
+
+Do not enable WAF for the system services with the following namespace prefixes as doing so may result in an undesired cluster state:
```

Comment:
Are there more system services besides those that start with these prefixes?

Consider:

Enabling WAF for (certain) system services may result in an undesired cluster state. Do not enable WAF for system service with the following prefixes:

Reply:
Yes, there could be more such services.

BTW, I'm changing the text as per your suggestion.